media-gfx/iscan/iscan-3.62.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools desktop flag-o-matic udev xdg-utils
+
+DESCRIPTION="EPSON Image Scan v3 for Linux"
+HOMEPAGE="https://support.epson.net/linux/en/imagescanv3.php"
+SRC_URI="https://support.epson.net/linux/src/scanner/imagescanv3/common/imagescan_${PV}.orig.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="gui"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-libs/boost:=
+	media-gfx/sane-backends
+	media-libs/tiff
+	virtual/libusb:1
+	virtual/jpeg
+	gui? ( dev-cpp/gtkmm:2.4 )
+	|| ( media-gfx/imagemagick[cxx] media-gfx/graphicsmagick[cxx] )
```

Comment:
It is incorrect to drop `graphicsmagick` USE flag and `:=` operator for `media-gfx/imagemagick` and `media-gfx/graphicsmagick`.
If you intend to make dependency on them unconditional, then you need:
```
graphicsmagick? ( media-gfx/graphicsmagick:=[cxx] )
!graphicsmagick? ( media-gfx/imagemagick:=[cxx] )
```

Reply:
imagemagick or graphicsmagick become hard dependency (as explained above), so there is no need for imagemagick/graphicsmagick use flags. We need either of them with cxx.

Reply:
You do not understand purpose of `graphicsmagick` USE flag.
It is similar to `libressl` USE flag.
`imagemagick` USE flag enables **optional** support for ImageMagick **OR** GraphicsMagick.
`graphicsmagick` USE flag is used to select GraphicsMagick **instead of** ImageMagick.
`ssl` USE flag enables **optional** support for OpenSSL **OR** LibreSSL.
`libressl` USE flag is used to select LibreSSL **instead of** OpenSSL.

Reply:
I see now. You want to forcefully specify "magick" implementation to use by graphicsmagick use flag.